src/runtime/contrib/tensorrt/tensorrt_builder.cc

```diff
@@ -185,6 +185,17 @@ TensorRTEngineAndContext TensorRTBuilder::BuildEngine() {
   return {engine, context, network_input_names_, network_output_names_, device_buffers};
 }
 
+void TensorRTBuilder::CreateDeviceBuffers(TensorRTEngineAndContext* engine_and_context) {
```

Comment:
The code in this function is a duplicate of the code in `BuildEngine()` - can you call this new function from BuildEngine to avoid the duplication?

Reply:
CreateDeviceBuffers is used in BuildEngine